apidoc/Titanium/UI/WebView.yml

```diff
@@ -35,23 +35,28 @@ description: |
     [evalJS](Titanium.UI.WebView.evalJS) method to execute a JavaScript expression
     inside the web view and retrieve the value of an expression.
 
+    <b>Android</b>:
     You can inject the local `Ti.App.fireEvent` bindings yourself by adding a script element using
-    evalJS.
+    [injectCode](Titanium.UI.WebView.injectCode) or manually with evalJS:
 
     ```js
+    // manual version
+    var code = '' // add binding.min.js code in here.
     webview.evalJS(
       'javascript=(function addBinding(){' +
         'var s=document.createElement("script");' +
         's.setAttribute("type","text/javascript");' +
-        's.innerHTML="' + require('/binding.js') + '";'
+        's.innerHTML="' + code + '";'
         +
         'document.getElementsByTagName("body")[0].appendChild(s);' +
       '})()'
     );
     ```
-    The `binding.min.js` is available in the [repository](https://github.com/tidev/titanium_mobile/tree/master/android/modules/ui/assets/Resources/ti.internal/webview).
+    The `binding.min.js` file is available in the [repository](https://github.com/tidev/titanium_mobile/tree/master/android/modules/ui/assets/Resources/ti.internal/webview).
 
-    For iOS check the example in <Titanium.UI.WebView.addScriptMessageHandler>.
+    <b>iOS</b>:
+    For iOS have a look at the example in <Titanium.UI.WebView.addScriptMessageHandler> for custom handler names or if you run the manual evalJS version
```

Comment:
Parity would be good. I just didn't know how to integrate the `injectEventBindings()` in ObjectivC :blush:  That's why I wanted to keep the evalJS example since that is working on both platforms. `window.webkit.messageHandlers.yourHandlerName` is not available in the Android webview so we can't directly map that. 

The default Ti way to communicate from webview -> app is still `Ti.App.fireEvent` and that is cross-platform. The docs were just always wrong saying that you can't use it with external URL, you could always inject that code with evalJS (or if it is your external page: just put the file on your server and load it right away) :smile: 

I'll try to see if I can add the iOS part!

Reply:
Wow, this PR is looking good! We can dive into the iOS stuff during our meeting. Great job!